### ISSUE TITLE
Not all songs can be downloaded from Spotify

### ISSUE THREAD

**Issue by Bakedhoven** [AllToMP3/alltomp3-app#49]
Just used this program for the first time and I have to say that I am very pleased! The only problem that I have with it, is that it is unable to download every song that I link from Spotify. I am unsure of why this might be happening as I download by the album and in some cases the whole album will have no dramas downloading and in others, the album might only partially be downloaded or not at all. I have tried searching Google for a fix to this problem, but it seems there is no solution as of yet. Please fix this and I would be happy to donate or even purchase a license to the program if it was necessary.

**Comment by vityou** [comment]
I haven't looked at the source code, but I'm guessing that it looks for the songs in your Spotify playlist on youtube or somewhere else and downloads them from there.  So some of the more obscure ones won't be there.

**Comment by thexbasic** [comment]
Thats true, and also the reason the quality cant be really 320 kbps, since YouTube uses 192 kbps at max, and even less for older or low quality videos... I guess Spotify wont easily allow people to download the real, unencrypted songs without any kind of restraint.. and theyre putting many resources in so it stays like this.
